src/workflow/workflow.ts

```diff
@@ -1,31 +1,31 @@
-import { IPkName, IRepositoryWritable } from "../repository/repository";
+import { IPkName, IRepositoryWritable } from "../repository/repository"
 
 export interface Context {
   source: string;
-  data: any;
+  data: {record: any, additionnal?: any};
```

Comment:
rename data to ActionContext

Reply:
fixed by [9dec683](https://github.com/all41-dev/server/pull/37/commits/9dec6838e68d41361af6eb59aa00d23023ba2105)